src/_tests/integration/import-cohort-data.integration-test.js

```diff
@@ -0,0 +1,30 @@
+const { findInGrid, exactClick, click, findIframe, findText, select } = require('./integration-utils')
+const dataExplorer = require('./data-explorer-utils')
+
+
+jest.setTimeout(10000)
```

Comment:
I think this might be unnecessary if we're specifying timeouts in individual tests.